src/index.json

```diff
@@ -298,6 +298,63 @@
                 "sha256Digest": "1faecf9d3b5955829e00b228e174fa6bf1ba327f0d64c2edfc9d096d2926e8dc"
             }
         ],
+        "azure-devops-extension": [
+            {
+                "downloadUrl": "https://azuredevopscli.blob.core.windows.net/releases/azure_devops_extension-1.0.0rc1-py2.py3-none-any.whl",
+                "filename": "azure_devops_extension-1.0.0rc1-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.minCliCoreVersion": "2.0.52",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 2",
+                        "Programming Language :: Python :: 2.7",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.4",
+                        "Programming Language :: Python :: 3.5",
+                        "Programming Language :: Python :: 3.6",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "VSTS_Social@microsoft.com",
+                                    "name": "Microsoft",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Microsoft/vsts-cli/tree/azuredevopscli-dev"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "azure-devops-extension",
+                    "run_requires": [
+                        {
+                            "requires": [
+                                    "python-dateutil==2.7.3",
+                                    "vsts==0.1.20",
+                                    "keyring~=13.2.1"
+                            ]
+                        }
+                    ],
+                    "summary": "Azure DevOps Command Line Interface",
```

Comment:
Please re-word your summary per: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md
Suggest something along the lines of: "Tools for managing Azure DevOps."